src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -433,8 +433,6 @@ void mp::QemuVirtualMachine::on_restart()
 
 void mp::QemuVirtualMachine::ensure_vm_is_running()
 {
-    QCoreApplication::processEvents();
```

Comment:
Shouldn't the `processEvents` in the second argument of `wait_until_ssh_up` and `wait_for_cloud_init` also be removed?

Reply:
@ricab,

No...we still want a way for the backend to pass in a special function for handling things it deems necessary while checking if `ssh` is ready.  In this particular case, we still want to check that the `QProcess` that starts `qemu-system-x86_64` has not exited for some reason while looping for the checking for `ssh`.

Reply:
After discussion, yes, those two function definitions call `processEvents` by default.  Thanks for pointing that out:)

I pushed the changed.  Thanks!